### ISSUE TITLE
no partition is assigned warnings

### ISSUE THREAD

**Issue by JonathonO** [snowflakedb/snowflake-kafka-connector#701]
Snowflake Sink v2.0.0

A Sink (call it Sink Z) was previously configured to consume from multiple topics (call them topic A, B and C). After some time and data being consumed from the 3 topics, Topic B is removed from the Sink configuration.

It remains in the Consumer Group for Sink Z without having it's partition assigned to any Sink Z Task. This causes constant warnings in logs: `[SF_KAFKA_CONNECTOR] no partition is assigned`.

There's no side effects other than constant warnings (approx. 9-10k in 24 hours for 1 Sink with 1 Topic unassigned).

**Comment by sfc-gh-japatel** [comment]
This might be because
1. We had a bug in our closing logic which cleared the entire map. It was fixed in release  https://github.com/snowflakedb/snowflake-kafka-connector/releases/tag/v2.0.1

Do you mind upgrading your connector and verifying this behavior?

**Comment by JonathonO** [comment]
> This might be because
> 
> 1. We had a bug in our closing logic which cleared the entire map. It was fixed in release  https://github.com/snowflakedb/snowflake-kafka-connector/releases/tag/v2.0.1
> 
> Do you mind upgrading your connector and verifying this behavior?

@sfc-gh-japatel of course! It might not be until next week now but we'll try that and get back to you 👍

**Comment by sfc-gh-akowalczyk** [comment]
Fixed in release 2.0.1
